ipams/builtin/builtin_unix.go

```diff
@@ -41,3 +46,13 @@ func Init(ic ipamapi.Callback, l, g interface{}) error {
 
 	return ic.RegisterIpamDriverWithCapabilities(ipamapi.DefaultIPAM, a, cps)
 }
+
+// SetDefaultIPAddressPool stores default address pool to local variable.
```

Comment:
Nit: SetDefaultIPAddressPool stores default address pool  (Its not a local variable)